packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.stories.ts

```diff
@@ -100,12 +103,19 @@ const formatDescription = `Configures the way that the numeric value is formatte
     <ul>
         <li>\`default\`: Integers are shown with no trailing zeros, the value is limited to 6 digits, and exponential notation is used for numbers that are large (\`>= 1e6\`) or small (\`< 1e-3\`) in magnitude.
         </li>
+        <li>\`decimal\`: Values as are formatted as decimal values, always displaying \`decimal-digits\` digits after the separator and never displaying exponential notation.
+        </li>
         <li>\`roundToInteger\`: Values are rounded to the nearest whole number. Exponential notation is never used. It can only safely represent integers up to the magnitude of \`MAX_SAFE_INTEGER\`.
         </li>
     </ul>
 </details>
 `;
 
+const validityDescription = `Readonly object of boolean values that represents the validity states that the column's configuration can be in.
+The object's type is \`TableColumnValidity\`, and it contains the following boolean properties:
+-   \`invalidDecimalDigits\`: \`true\` when \`format\` is configured to \`decimal\` and \`decimal-digits\` is set to a number less than 0 or greater than 20.
+`;
+
 const alignmentDescription = `Configures the alignment of the value within the column.
```

Comment:
Reminder to update the "Default Alignment" section to mention the behavior of the decimal format.